tools/test-proxy/Azure.Sdk.Tools.TestProxy/Transforms/StorageRequestIdTransform.cs

```diff
@@ -15,12 +15,12 @@ public StorageRequestIdTransform(ApplyCondition condition = null)
             Condition = condition;
         }
 
-        public override void ApplyTransform(HttpRequest request, HttpResponse response)
```

Comment:
Are StorageRequestIdTransform and ApiVersionTransform the same just a different header name?

Reply:
Looks like it. I think ClientIdTransform is the same as well but there appears to be a bug in that it is adding to the request header instead of response header. We should probably try to generalize into some a header transform that copies from request onto response. This is slightly different from the transform I am adding.

Reply:
We can't really use ApplyCondition because we wouldn't know what the header value is.

Reply:
I prototyped exposing the matched request value.